### ISSUE TITLE
Scoring accuracy

### ISSUE THREAD

**Issue by sid-dinesh94** [Ninjani/Data-Mining-Project#2]
We don't need to have a filled out `y_test_tags` in the test_reducer. Instead we can keep a list or dictionary of sets of tags for each instance and check for set membership in our accuracy function. This might be faster @quizywiz @Ninjani

**Comment by Ninjani** [comment]
yeah, we already discussed this no?

**Comment by sid-dinesh94** [comment]
Yeah. Opened the issue so we dont forget to implement this. Mark as complete when done.
